app/messages/Messages.scala

```diff
@@ -1187,11 +1187,21 @@ object Messages {
       welsh   = s"${if (hasUpfrontPayment) "Caiff eich taliad ymlaen llaw ei gymryd cyn pen 10 diwrnod gwaith. " else ""}Caiff eich taliad nesaf ei gymryd ar $paymentDate neu’r diwrnod gwaith nesaf."
     )
 
+    val `Your tax account will be updated with your payment plan within 24 hours.`: Message = Message(
+      english = "Your tax account will be updated with your payment plan within 24 hours.",
+      welsh   = "Bydd eich cyfrif treth yn cael ei ddiweddaru gyda'ch cynllun talu cyn pen 24 awr."
```

Comment:
is that a straight quote in the welsh?

Reply:
Oh nice spot, I'll update to a smart quote

Reply:
Updated